### PR TITLE
Conditional session start

### PR DESCRIPTION
**Pull request by meminens**
Start new session if it hasn't already been started

Fixes unintentional signouts

### CODE REVIEW

File: php/index.php [hunk 1]
```diff
@@ -36,8 +36,11 @@
 // Check if a function has been specified
 if (!empty($fn)) {
 
-	// Start the session and set the default timezone
-	session_start();
+	session_name('lychee');
```
Comment (ildyria) [inline php/index.php]: Is this `session_name` really needed?

**Review by ildyria — CHANGES_REQUESTED** [review]
Please check the change I made with respect to PHP >= 5.4.0 if it still works. If it does, I will merge.

File: php/index.php [hunk 2]
```diff
@@ -36,11 +36,11 @@
...
 if (!empty($fn)) {
 
-	session_name('lychee');
-        // Start the session and set the default timezone
-        if (session_id() == '') {
-            session_start();
-        }
+	// session_name('lychee');
+    // Start the session and set the default timezone
+    if (session_status() == PHP_SESSION_NONE) {
```
Comment (ildyria) [inline php/index.php:41]: I mean that change.

File: php/index.php (hunk 1 above)
Comment (meminens) [reply php/index.php]: Perhaps not. Might be useful for server configuration in which `session.auto_start` is not set to `True` in php.ini.

File: php/index.php (hunk 2 above)
Comment (meminens) [reply php/index.php:41]: Tested on my server and it works well.

`PHP 7.2.10` & `Apache 2.4.29`

**Review by ildyria — APPROVED** [review]

**Comment by meminens** [comment]
Thanks for the merge!

**Pull request merged**